doc/06-distributed-monitoring.md

```diff
@@ -755,6 +755,12 @@ the [configuration modes](06-distributed-monitoring.md#distributed-monitoring-co
 
 ### Agent Setup on Windows <a id="distributed-monitoring-setup-agent-windows"></a>
 
+!!! important
+
+    [Icinga for Windows](https://icinga.com/docs/icinga-for-windows/latest/doc/000-Introduction/)
+    is the recommended way to install, setup and update Icinga 2 on Windows.
+    The following section describes the classic installer.
```

Comment:
I'd say "This section" here (referring to "Agent Setup on Windows" with its three subsections). "The following section" can be interpreted as "Agent Setup on Windows: Installer" but you shouldn't need the two (sub)sections following after that one with Icinga for Windows.

Maybe even say "classic installation procedure" instead of "classic installer" to make clear this is not only referring to the next section as this has "Installer" in its name.